### ISSUE TITLE
`.travis.yml` is not well formatted

### ISSUE THREAD

**Issue by Sahloul** [IntelVCL/Open3D#310]
I have been struggling to make a python package install in the Travis CI for some hours now!

The command `pyenv global 3.6` utilized to setup the Travis CI Linux-image seems to encapsulate Python inside a virtual environment. The problem is that CMake script cannot figure out the correct site directory to install `py3d.cpython-36m-x86_64-linux-gnu.so` python library.

However, I believe there is something wrong with the `.travis.yml` setting, and we should not use `pyenv global 3.6`. Another interesting thing is that `Ruby` image is loaded by this setting file, instead of Python, or GCC or similar things...

### How to reproduce
Both Python scripts utilized inside the CMake:
```python
import site

print([path for path in site.getsitepackages() if path.endswith('packages')][-1])

```
and
```python
from distutils import sysconfig

print(sysconfig.get_python_lib())
```
returns the virtualenv site-packages that is not gloabl site-package, and any installation to is does not get detected.

#### Actual output
/opt/pyenv/versions/3.6/lib/python3.6/site-packages

#### Expected output
/opt/python/3.6.3/lib/python3.6/site-packages

I even tried `echo "/opt/pyenv/versions/3.6/lib/python3.6/site-packages/" > /opt/python/3.6.3/lib/python3.6/site-packages/open3d.pth;` without any success.

#### Travis CI output
```
...
-- Installing: /opt/pyenv/versions/3.6/lib/python3.6/site-packages/py3d.cpython-36m-x86_64-linux-gnu.so
...
$ sudo python file_io.py
Traceback (most recent call last):
  File "file_io.py", line 5, in <module>
    from py3d import *
ImportError: No module named py3d
The command "sudo python file_io.py" exited with 1.
Done. Your build exited with 1.
```

**Comment by syncle** [comment]
I agree. `Pyenv` was some trick to give python 3.6 environment. It was okay until we reached to the installation issue. BTW, I think this is addressed in c071cd3. Am I correct?
